internal/backends/testing/search_api.rs

```diff
@@ -27,13 +27,23 @@ pub(crate) fn search_item(
     result
 }
 
+/// `ElementHandle`` wraps an existing element in a Slint UI. An ElementHandle does not keep
+/// the corresponding element in the UI alive. Use [`Self::is_valid()`] to verify that
+/// it is still alive.
+///
+/// Obtain instances of `ElementHandle` by querying your application through `[Self::find_by_accessible_label()`].
```

Comment:
tick inside the brackets

```suggestion
/// Obtain instances of `ElementHandle` by querying your application through
/// [`Self::find_by_accessible_label()`].
```